ansible_collections/arista/cvp/docs/schema/cv_device_v3.md

```diff
@@ -0,0 +1,15 @@
+# Schema for cv_device_v3
+
+| Variable | Type | Required | Default | Choices | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| apply_mode | str | No | loose | loose<br>strict | Set how configlets are attached/detached on device. If set to strict all configlets not listed in your vars are detached |
+| search_key | str | No | hostname | fqdn<br>hostname<br>serialNumber | Key name to use to look for device in Cloudvision |
+| state | str| No | present | present<br>factory_reset | Set if ansible should build or remove devices on Cloudvision |
```

Comment:
```suggestion
| state | str| No | present | present<br>factory_reset | Set if Ansible should build or remove devices on CloudVision |
```

Reply:
Added this change in another suggested change. Resolving this conversation